src/ucp/api/ucp.h

```diff
@@ -2805,6 +2805,8 @@ ucs_status_ptr_t ucp_am_send_nbx(ucp_ep_h ep, unsigned id,
  * @param [in]  count      Number of elements to receive into @a buffer.
  * @param [in]  param      Operation parameters, see @ref ucp_request_param_t.
  *
+ * @return NULL                 - The receive operation was completed
+ *                                immediately.
```

Comment:
do we need to change for TAG_RECV_NBX as well?
let's have all doc changes in separate PR

Reply:
ok will remove here